apinotes/CoreGraphics.apinotes

```diff
@@ -39,14 +43,269 @@ Functions:
   SwiftName: "CGRect.contains(self:_:)"
 - Name: CGRectIntersectsRect
   SwiftName: "CGRect.intersects(self:_:)"
-# The below are left alone, due to an unknown bug
-# FIXME: fix me
+# The below are not available in Swift
+- Name: CGColorSpaceCreateDeviceGray
+  Availability: nonswift
 - Name: CGColorSpaceCreateDeviceRGB
-  SwiftName: "CGColorSpaceCreateDeviceRGB()"
+  Availability: nonswift
+- Name: CGColorSpaceCreateDeviceCMYK
+  Availability: nonswift
+
+# The below are attempts at providing better names than inference
+# CGAffineTransform
+- Name: CGAffineTransformMakeTranslation
+  SwiftName: CGAffineTransform.init(withTranslationX:y:)
+- Name: CGAffineTransformMakeScale
+  SwiftName: CGAffineTransform.init(withScaleX:y:)
+- Name: CGAffineTransformMakeRotation
+  SwiftName: CGAffineTransform.init(withRotationAngle:)
+- Name: CGAffineTransformTranslate
+  SwiftName: CGAffineTransform.translateBy(self:x:y:)
+- Name: CGAffineTransformScale
+  SwiftName: CGAffineTransform.scaleBy(self:x:y:)
+- Name: CGAffineTransformRotate
+  SwiftName: CGAffineTransform.rotate(self:byAngle:)
+- Name: CGAffineTransformConcat
+  SwiftName: CGAffineTransform.concat(self:_:)
+- Name: CGAffineTransformEqualToTransform
+  SwiftName: CGAffineTransform.equalTo(self:_:)
+- Name: CGPointApplyAffineTransform
+  SwiftName: CGPoint.applyAffineTransform(self:_:)
+- Name: CGSizeApplyAffineTransform
+  SwiftName: CGSize.applyAffineTransform(self:_:)
+- Name: CGRectApplyAffineTransform
+  SwiftName: CGRect.applyAffineTransform(self:_:)
+# CGBitmapContext
+# FIXME: below doesn't work
+- Name: CGBitmapContextCreateImage
+  SwiftName: CGContextRef.makeImageFromBitmap(self:)
```

Comment:
It's due to a limitation in the ClangImporter, where name lookup is done on C names rather than Swift type names. The C name CGContext turns out to be a forward declaration only, and we can't import those as decl contexts.

I've asked @DougGregor if we should use Swift lookup instead.